data/json/terrain.json

```diff
@@ -235,6 +246,7 @@
         "type" : "terrain",
         "id" : "t_strconc_floor",
         "name": "reinforced concrete floor",
+        "description": "Extremely resilient floor made from carefully placed rebar and poured concrete, capable of providing protection from the elements.  Still requires supporting walls, otherwise it may very well cave in.",
```

Comment:
`otherwise it may very well cave in`

The floor may cave in? Is this accurate?